### ISSUE TITLE
Export types like TaskInstance

### ISSUE THREAD

**Issue by djmattyg007** [MartinMalinda/vue-concurrency#46]
This will provide more flexibility when typing things.

**Comment by MartinMalinda** [comment]
They're exported but maybe not in root, I'll take a look.

So far this should work I think: (change vue3 to vue2 if needed):

```ts
import { Task } from "vue-concurrency/dist/vue3/src/Task";
import { TaskInstance } from "vue-concurrency/dist/vue3/src/TaskInstance";
```

I'll try to make this work:

```ts
import { Task, TaskInstance } from "vue-concurrency";
```

**Comment by djmattyg007** [comment]
Importing from nested paths feels very clunky. It would definitely be easier if the relevant types were exported through the main entrypoint (something which typescript absolutely fully supports).

**Comment by MartinMalinda** [comment]
This should be fixed as part of `2.1.1`

<img width="676" alt="Screenshot 2021-07-18 at 12 18 51" src="https://user-images.githubusercontent.com/9468793/126063639-e687413b-2c15-4758-835a-2327c332d31b.png">
